src/test/scala/cognite/spark/v1/FlexibleDataModelsRelationTest.scala

```diff
@@ -130,7 +129,12 @@ class FlexibleDataModelsRelationTest extends FlatSpec with Matchers with SparkTe
                   .format(InstancePropertyValue.Timestamp.formatter)}' as timestampProp1,
                 |null as timestampProp2,
                 |'{"a": "a", "b": 1}' as jsonProp1,
-                |null as jsonProp2
+                |null as jsonProp2,
+                |named_struct(
+                |    'space', '$spaceExternalId',
+                |    'externalId', '$startNodeExtId'
+                |) as directRelation1,
```

Comment:
will this look like that in jetfire? or there will be a UDF for readability? @vuhainguyen

Reply:
in jetfire we can use the UDF yes, Spark DS users have to use this redundant syntax instead